src/test/java/org/gridsuite/modification/server/utils/NetworkCreation.java

```diff
@@ -334,14 +336,19 @@ private static void createLine(Network network, String id, String name, String v
 
     @SuppressWarnings("SameParameterValue")
     private static void createLoad(VoltageLevel vl, String id, String name,
-                                   int node, double p0, double q0) {
-        vl.newLoad()
+                                   int node, double p0, double q0, String feederName1, int feederOrder, ConnectablePosition.Direction direction) {
+        var l = vl.newLoad()
             .setId(id)
             .setName(name)
             .setNode(node)
             .setP0(p0)
             .setQ0(q0)
             .add();
+        l.newExtension(ConnectablePositionAdder.class)
+                .newFeeder1()
```

Comment:
use newFeeder() here
newFeeder1, newFeeder2, newFeeder3 are used for two and three windings transformers

Reply:
my bad ,nice catch